spec/lib/extensions/virtual_total_spec.rb

```diff
@@ -75,4 +380,38 @@ def model_with_children(count)
       end.reload
     end
   end
+
+  # Duplicated from VmOrTemplateSpec#provisioned_storage since this can't be
+  # simulated in SQLite, since they allow you having a ORDER BY with a column
```

Comment:
>since they allow you having a ORDER BY with a column

since they allow you **to have an** ORDER BY with a column

Reply:
Thanks, will change.